addons/mail/views/public_templates.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="mail.public_layout">&lt;!DOCTYPE html&gt;
+        <html class="h-100">
+            <head>
+                <meta charset="utf-8"/>
+                <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"/>
+                <title t-esc="title or 'Odoo'"/>
```

Comment:
can we use t-out ? (other var in this template use t-out)

Reply:
yes, we don’t need t-esc for a title indeed.